.repo-metadata.json

```diff
@@ -4,12 +4,13 @@
     "product_documentation": "https://cloud.google.com/iam/docs/audit-logging",
     "client_documentation": "https://cloud.google.com/python/docs/reference/iamlogging/latest",
     "issue_tracker": "https://github.com/googleapis/python-iamlogging/issues",
-    "release_level": "ga",
+    "release_level": "stable",
     "language": "python",
     "library_type": "OTHER",
     "repo": "googleapis/python-iam-logging",
     "distribution_name": "google-cloud-iam-logging",
     "api_id": "",
     "default_version": "v1",
-    "codeowner_team": ""
+    "codeowner_team": "",
+    "api_shortname": "iamlogging"
```

Comment:
This is an `OTHER` library, how is `api_shortname` decided in this case?

Reply:
Fixed.